signer/storage/aes_gcm_storage.go

```diff
@@ -129,7 +129,10 @@ func (s *AESEncryptedStorage) writeEncryptedStorage(creds map[string]storedCrede
 	return nil
 }
 
-func encrypt(key []byte, plaintext []byte) ([]byte, []byte, error) {
+// encrypt encrypts plaingtext with the given key, with additionaldata
```

Comment:
There is one g too much in plaintext and one space missing between additional and data

Reply:
fixed